main.cpp

```diff
@@ -195,26 +249,36 @@ Thing 2) airport
    
 struct Airport
 {
-    //number of planes
+    
     int numberOfPlanes = 10;
-    //number of floor staff
     int numberOfFloorStaff = 100;
-    //number of aeroplane engineers
     int numberOfAeroPlaneEngineers = 50;
-    //number of runways
     int numberOfRunways = 5;
-    //number of radar
     int numberOfRAdar = 1;
 
-    //land and takeoff planes
     void landAndTakeOffPlanes(bool hasConfirmation = true);
-    //centralise travel departure
-    void centraliseTravelDeparture(); 
-    //service travel industry
-    void serviceTravelIndustry(bool notOvercapacity);
+    void centraliseTravelDeparture(bool departure = true); 
+    void serviceTravelIndustry(bool notOverCapacity);
 
 };
 
+void Airport::landAndTakeOffPlanes(bool hasConfirmation)
+{
+    (hasConfirmation=true);
+}
+
+void Airport::centraliseTravelDeparture(bool departure)
+{
+    (departure = true);
+}
+
+void Airport::serviceTravelIndustry(bool notOverCapacity) 
+{
+    (notOverCapacity = true);
+}   
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise

Reply:
Removed parenthesis